regression-test/test_2160.py

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
Use https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/test_fasttext.py for this test